semgrep/tests/e2e/rules/long.yaml

```diff
@@ -7,10 +7,6 @@ rules:
       ...
       <... $B ...>
       ...
```

Comment:
This is what we can handle now ... just 2 deep-expr with a metavariable. This is because before the $B itself was matching greedily, but we actually need to match all possible subexpressions.